### PR TITLE
feat: support for snx

### PR DESCRIPTION
**Pull request by dohaki**
Closes ACX-1065 and ACX-1066

Adds support for SNX.

Note: I added SNX to the `DISABLED_TOKENS` env var on production, so this PR can be merged and tested with the preview link.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "license": "AGPL-3.0-only",
   "dependencies": {
-    "@across-protocol/sdk-v2": "^0.5.5",
+    "@across-protocol/sdk-v2": "^0.5.6",
```
Comment (james-a-morris) [inline package.json:7]: Nit: there's a more recent version if we'd want to upgrade to the latest SDK package

**Review by james-a-morris — APPROVED** [review]
Left a nit

File: package.json (hunk 1 above)
Comment (dohaki) [reply package.json]: Good point. Bumped here https://github.com/across-protocol/frontend-v2/pull/741/commits/6161b6e13499683a2563865842eec1342e126d7e

Comment (dohaki) [reply package.json]: Nooo 😞 We encountered this issue again with the latest SDK version 
```
Creating an optimized production build...
Failed to compile.

./node_modules/@uma/financial-templates-lib/node_modules/@uma/common/dist/HardhatConfig.js
Cannot find module: 'hardhat-tracer'. Make sure this package is installed.

You can install this package by running: yarn add hardhat-tracer.


error Command failed with exit code 1.
```
I will revert the commit for now until this issue is fixed upstream.

**Pull request merged**